crates/y-sweet-core/src/api_types.rs

```diff
@@ -27,13 +22,12 @@ impl Authorization {
 
 #[derive(Deserialize)]
 #[allow(unused)]
+#[serde(default)]
```

Comment:
I believe this means if someone passes an unknown field (e.g. the old `metadata` field), deserialization will still succeed.